polkadot/node/primitives/src/disputes/status.rs

```diff
@@ -26,7 +26,7 @@ pub type Timestamp = u64;
 ///
 /// NOTE: This status is persisted to the database, any changes have to be versioned and a db
 /// migration will be needed.
-#[derive(Debug, Clone, Copy, Encode, Decode, PartialEq)]
+#[derive(Debug, Clone, Copy, Encode, Decode, PartialOrd, Ord, PartialEq, Eq)]
```

Comment:
why do we need to Order by DisputeStatus if it's used as a value in BTreeMap - it also doesn't make sense logically